lua/lvim/core/bufferline.lua

```diff
@@ -58,7 +58,9 @@ M.config = function()
     options = {
       mode = "buffers", -- set to "tabs" to only show tabpages instead
       numbers = "none", -- can be "none" | "ordinal" | "buffer_id" | "both" | function
-      close_command = "bdelete! %d", -- can be a string | function, see "Mouse actions"
+      close_command = function(bufnr)
+        M.buf_kill("bd", bufnr, true)
+      end, -- can be a string | function, see "Mouse actions"
```

Comment:
```suggestion
      close_command = function(bufnr) -- can be a string | function, see "Mouse actions"
        M.buf_kill("bd", bufnr, false)
      end,
```

Reply:
I don't think the close button will even appear on a modified buffer anyway, but it's always good to prevent data loss :smile: